### PR TITLE
Fixing timeout issue with few NIC drivers during ethtool …

### PR DESCRIPTION
**Pull request by manvanthar**
…test.

Fixing timeout issue with few NIC drivers during ethtool test.

Signed-off-by: Manvanthara Puttashankar <manvanth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
@vaishnavibhat review please.

File: io/net/ethtool_test.py [hunk 1]
```diff
@@ -70,6 +71,16 @@ def interface_state_change(self, interface, state, status):
         cmd = "ip link set dev %s %s" % (interface, state)
         if process.system(cmd, shell=True, ignore_status=True) != 0:
             return False
+        if state == "up":
+            for _ in range(0, 120, 10):
```
Comment (abdhaleegit) [inline io/net/ethtool_test.py]: Can we use wait module in avocado utils to wait for function to return true 
wait.wait_for(check_state(interface), 30)

as per discussion with murilo a 30 sec wait is sufficient for any adapter link to come up

Comment (manvanthar) [reply io/net/ethtool_test.py]: @abdhaleegit this has been tested...  30 sec is an random assumtions.  tg3 driver takes from 20 sec to 2 min to complete based on what ethtool parameter is changed.

Comment (manvanthar) [reply io/net/ethtool_test.py]: btw wait module is not fully functional!!. so cant use it.

Comment (manvanthar) [reply io/net/ethtool_test.py]: the logic checks the status of interface from 10 sec to 2 min. so there is no full wait of the system indefintly or some random 30 sec waiting .

Comment (manvanthar) [reply io/net/ethtool_test.py]: Abdul,  Let me know if you ok with the justification

Comment (narasimhan-v) [reply io/net/ethtool_test.py]: @manvanthar Also, if this is a generic solution, can this be pushed to avocado utils ?

@abdhaleegit your comments ?

Comment (abdhaleegit) [reply io/net/ethtool_test.py]: the wait.wait_for is used to wait for command to complete .. but the logic here is really waiting till the network link is up.. so we can go ahead with this logic.. in future a avocado util to check link up will be better.

Comment (abdhaleegit) [reply io/net/ethtool_test.py]: we can push this till than

**Comment by narasimhan-v** [comment]
@abdhaleegit Thanks for ack. Merging.

**Pull request merged**